### PR TITLE
Animations Tutorial: Change event handler param

### PR DESCRIPTION
**Pull request by DannyHellbane**
Hey guys, just another humble edit.  I noticed these were correct in a previous iteration of the tutorial but didn't seem to survive the zombies.

### CODE REVIEW

**Comment by agubler** [comment]
Hey @DannyHellbane, it is actually correct with the RC release of widgets - The widgets have been renamed `index.ts` so that consumers can just import from `@dojo/widgets/slider`.

See the source: https://github.com/dojo/widgets/tree/master/src/slider

**Comment by DannyHellbane** [comment]
I'm in two minds opening this pull request because last time @tomdye confirmed that some unexpected features in dojo 2 tutorials are deliberate design choices to make things interesting.  It's proving to be a brilliant tactic because, in this case, you could argue the current tutorial behaviour showcases Typescript's strong type advantages rather effectively.

Not super-psyched to deprive others of the fun.  So if this is one of those design features please feel free to delete this pull request.

**Comment by DannyHellbane** [comment]
Ok. Got it. Thanks. Sorry.

**Comment by tomdye** [comment]
@DannyHellbane you got that that was meant as a joke right?

**Comment by DannyHellbane** [comment]
Sorry, @tomdye but this is where we are, man: [Family Guy: We live in the post joke world](https://www.youtube.com/watch?v=2PEHFkfZymo)

File: site/source/tutorials/008_animations/demo/finished/zombies/src/widgets/Zombies.ts [hunk 1]
```diff
@@ -153,7 +153,8 @@ export class Zombies extends WidgetBase {
 		return hearts;
 	}
 
-	private _onZombieLegsPlaybackRateChange(value: string) {
+	private _onZombieLegsPlaybackRateChange(event: Event) {
```
Comment (agubler) [inline site/source/tutorials/008_animations/demo/finished/zombies/src/widgets/Zombies.ts]: The interface is `value: string` with the latest `Slider` version, see https://github.com/dojo/widgets/blob/master/src/common/interfaces.d.ts#L45

Comment (DannyHellbane) [reply site/source/tutorials/008_animations/demo/finished/zombies/src/widgets/Zombies.ts]: @agubler  Yep, please abort this pull request, the embarrassment is killing me

Comment (agubler) [reply site/source/tutorials/008_animations/demo/finished/zombies/src/widgets/Zombies.ts]: @DannyHellbane okay!

**Comment by agubler** [comment]
Closing as the currently reflects the most recent version of @dojo/widgets - Thanks @DannyHellbane for raising a PR regardless 😄

**Pull request closed without merging**